### DISCUSSION TITLE
position

### DISCUSSION THREAD

**Discussion by KaliStudio** [YimMenu/YimMenu#2446] [Q&A]
Hello, I would like to know how to modify the starting position of the overlay/menu?

**Comment by lonelybud** [comment]
i dont think there is such a setting in menu such as. You have to manually hard code it. 

For starting reference check these files -

`src\gui.cpp`
`src\views\core\view_root.cpp`

In your case this is the file `src\views\core\view_overlay.cpp`.

**Reply by Yimura** [reply]
The menu isn't really setup in a way to allow for that sort of customization sadly. Unless you go into the source code and modify it yourself.

**Reply by KaliStudio** [reply]
I would like to modify the starting position so as not to hide the game bubbles
